CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ set(PROJECT_DESCRIPTION
 set(PROJECT_URL
         "https://github.com/rawrtc/rawrtc")
 
+# Use prefix pkgconfig
+set(ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig/:$ENV{PKG_CONFIG_PATH}")
```

Comment:
That's pretty neat! Technically, it would allow us to drop the section in the readme where the [pkg-config path is being set to the prefixed directory](https://github.com/rawrtc/rawrtc#package-configuration-path) but I think it's just fine the way it is.